arch/eventloop_posix_tcp.c

```diff
@@ -490,6 +490,66 @@ TCP_registerListenSocketDomainName(UA_ConnectionManager *cm, const char *hostnam
     return UA_STATUSCODE_GOOD;
 }
 
+#ifdef _WIN32
+static
+UA_RegisteredFD *
```

Comment:
In all the other functions, we say
```c
static UA_RegisteredFD *
findRegisteredFD(TCPConnectionManager *cm, uintptr_t connectionId) {
```

Please keep with the established code formatting.